gradescope-to-db/src/main.rs

```diff
@@ -107,20 +107,21 @@ async fn insert_regrade(
     regrade: &Regrade,
 ) -> Result<()> {
     let mut db = db_pool.acquire().await?;
-    let (assignment_id, student_name, question_number, question_title, grader_name) = (
+    let (assignment_id, student_name, question_number, question_title, grader_name, completed) = (
         assignment.id(),
         regrade.student_name().as_str(),
         regrade.question_number().as_str(),
         regrade.question_title().as_str(),
         regrade.grader_name().as_str(),
+        i8::from(regrade.completed()),
     );
 
     sqlx::query!(
         "
-        INSERT OR IGNORE INTO regrade (assignment_id, student_name, question_number, question_title, grader_name)
-        VALUES (?, ?, ?, ?, ?);
+        INSERT OR IGNORE INTO regrade (assignment_id, student_name, question_number, question_title, grader_name, completed)
```

Comment:
I'm not sure how to fix it (a brief search a while ago made it look nontrivial with SQLite), but this won't actually update as regrades are completed. Not necessarily a problem here, but something to be aware of.
```suggestion
        INSERT ?? regrade (assignment_id, student_name, question_number, question_title, grader_name, completed)
```

Reply:
You probably want to add a unique index on `(assignment_id, student_name, question_number)`, and then change the query to be
```sql
INSERT INTO regrade (assignment_id, student_name, question_number, question_title, grader_name)
VALUES (?, ?, ?, ?, ?)
ON CONFLICT (assignment_id, student_name, question_number) DO UPDATE
SET completed = excluded.completed; -- and we can update the other columns here if we want
```
It would probably be better to make it a little more clear if these inserts are actually upserts, and we can do that through another PR.